_data/sites.json

```
@@ -10624,6 +10624,16 @@
         ]
     },
 
+    {
+        "name": "Lifesum",
+        "url": "https://lifesum.com/account/delete-account",
+        "difficulty": "easy",
+        "notes": "On you account page, go to the ['Close Account & Delete Data' tab](https://lifesum.com/account/delete-account).",
```

```suggestion
        "notes": "On you account page, go to the 'Close Account & Delete Data' tab.",
```
We generally only add links to the notes when they are different from the URL